### PR TITLE
Honor -persist flag in kind cluster destruction

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

File: acceptance/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -269,6 +272,8 @@ func AddStepsTo(sc *godog.ScenarioContext) {
 
 func InitializeSuite(ctx context.Context, tsc *godog.TestSuiteContext) {
 	tsc.AfterSuite(func() {
-		kind.Destroy(ctx)
+		if !persisted {
```
Comment (zregvart) [inline acceptance/kubernetes/kubernetes.go:275]: So we don't need the global variable

```suggestion
		if !testenv.Persisted(ctx) {
```

**Review by zregvart — COMMENTED** [review]
I don't think we need the global variable

File: acceptance/kubernetes/kubernetes.go (hunk 1 above)
Comment (lcarva) [reply acceptance/kubernetes/kubernetes.go:275]: I don't think this instance of context will have the right info. This is the "default" context. Let me try it.

Comment (lcarva) [reply acceptance/kubernetes/kubernetes.go]: It worked! Thank you.

**Review by zregvart — APPROVED** [review]

**Pull request merged**